examples/index.js

```diff
@@ -43,6 +43,50 @@ const Description = React.createClass({
 
 })
 
+const PostCard = React.createClass({
+
+  propTypes: {
+    post: T.shape({
+      author: T.shape({
+        first_name: T.string.isRequired,
+        last_name: T.string.isRequired,
+      }).isRequired,
+
+      excerpt: T.string.isRequired,
+      likeCount: T.number.isRequired,
+      title: T.string.isRequired,
+      onLikeClick: T.func.isRequired,
+    }).isRequired,
+
+    showLikes: T.bool,
+  },
+
+  render() {
+    const { post, showLikes } = this.props
+    const { author, excerpt, likeCount, title, onLikeClick } = post
+
+    return (
+      <div
+        style={{
+          padding: 20,
+          width: 300,
+          background: 'white',
+        }}
+      >
+        <h2>{title}</h2>
+
+        <p style={{ color: 'gray', textAlign: 'right' }}>
+          <small>By {author.first_name} {author.last_name}</small>
+        </p>
+
+        <p>{excerpt}</p>
+        {showLikes && <p onClick={onLikeClick}>Likes: {likeCount}</p>}
+      </div>
+    )
+  },
+
+})
```

Comment:
I think there is no need for such complex example here, the most minimal one that only shows nesting will do (one or two props).